rcl_lifecycle/src/default_state_machine.c

```diff
@@ -690,10 +690,27 @@ rcl_lifecycle_init_default_state_machine(
 
   return ret;
 
-fail:
+// Semicolon handles the "a label can only be part of a statement..." error
+fail:;
```

Comment:
```suggestion
fail:
```

Or was that semicolon intended?

Reply:
The compiler wouldn't let me declare `current_error` directly after `fail:` without it. The semicolon effectively creates a new statement. Should I move the variable declarations to the top?